ROM v49 cannot be built: ld65: Invalid definition: '__CHARLAE_LOAD__='

I am trying this on a fresh github pull, cc65 v 2.19, OSX Sonoma 14.6, Apple M3. Lzsa v1.4.1, Python v3.9.6. Exact command line that produced the error is below.  The problem variable appears to be a valid hex number.

`git diff --quiet && /bin/echo -n $( (git rev-parse --short=8 HEAD || /bin/echo "00000000") | tr '[:lower:]' '[:upper:]') > build/x16/../signature.bin \
        || /bin/echo -n $( /bin/echo -n $(git rev-parse --short=7 HEAD || echo "0000000") | tr '[:lower:]' '[:upper:]'; /bin/echo -n '+') > build/x16/../signature.bin
ld65 -C build/x16/cfg/kernal-x16.cfg build/x16/kernal/declare.o build/x16/kernal/vectors.o build/x16/kernal/kbdbuf.o build/x16/kernal/memory.o build/x16/kernal/lzsa.o build/x16/kernal/signature.o build/x16/kernal/cbm/editor.o build/x16/kernal/cbm/channel/channel.o build/x16/kernal/cbm/init.o build/x16/kernal/cbm/memory.o build/x16/kernal/cbm/nmi.o build/x16/kernal/cbm/irq.o build/x16/kernal/cbm/util.o build/x16/kernal/cbm/serial.o build/x16/kernal/x16/extapi.o build/x16/kernal/x16/65c816/interrupt.o build/x16/kernal/x16/65c816/stack.o build/x16/kernal/x16/65c816/extapi16.o build/x16/kernal/drivers/x16/x16.o build/x16/kernal/drivers/x16/memory.o build/x16/kernal/drivers/x16/screen.o build/x16/kernal/drivers/x16/ps2data.o build/x16/kernal/drivers/x16/ps2kbd.o build/x16/kernal/drivers/x16/ps2mouse.o build/x16/kernal/drivers/x16/joystick.o build/x16/kernal/drivers/x16/clock.o build/x16/kernal/drivers/x16/framebuffer.o build/x16/kernal/drivers/x16/sprites.o build/x16/kernal/drivers/x16/entropy.o build/x16/kernal/drivers/x16/beep.o build/x16/kernal/drivers/x16/i2c.o build/x16/kernal/drivers/x16/smc.o build/x16/kernal/drivers/x16/rtc.o build/x16/kernal/drivers/generic/softclock_timer.o build/x16/kernal/graph/graph.o build/x16/kernal/fonts/fonts.o build/x16/kernal/graph/console.o build/x16/kernal/graph/palette.o build/x16/kernal/ieee_switch.o -o build/x16/kernal.bin -m build/x16/kernal.map -Ln build/x16/kernal.sym \
        -D __CHARLAE_LOAD__=$EE00 -D __CHARLAE2_LOAD__=$F100 -D __CHARPET_LOAD__=$C000 -D __CHARPET2_LOAD__=$D000 \ \
        -D __CHARISO2_LOAD__=$D800 -D __CHARKAT_LOAD__=$F400 -D __CHARISO_LOAD__=$C800 -D __CHARCYR_LOAD__=$E800 \ \
        -D __CHARANSI_LOAD__=$E000 -D __CHARCYR2_LOAD__=$EB00 \
ld65: Invalid definition: '__CHARLAE_LOAD__='
make: *** [build/x16/kernal.bin] Error 1`

--- a/Makefile
+++ b/Makefile
@@ -333,7 +333,7 @@ GIT_SIGNATURE=$(BUILD_DIR)/../signature.bin
 # Wrapper for findsymbols that escapes the output and guards against non-zero exit codes
 define findsymbols
 $(strip \
-	$(subst $,\$,$(shell ${BUILD_DIR}/../../findsymbols $(1) $(2))) \
+	$(shell ${BUILD_DIR}/../../findsymbols $(1) $(2)) \
 	$(if \
 		$(filter 0,$(firstword $(.SHELLSTATUS) 0)),, \
 		$(error findsymbols failed with exit code $(.SHELLSTATUS)) \

--- a/findsymbols
+++ b/findsymbols
@@ -56,7 +56,7 @@ BEGIN {
 {
 	for (s in symbols) {
 		if (symbols[s] == substr($3,2)) {
-			output[symbols[s]] = "$" substr($2,3)
+			output[symbols[s]] = "\\$" substr($2,3)
 		}
 	}
 }

